### PR TITLE
Implemented code to parse csv file using papaparser package (#1vd3pgk)

### PR DESCRIPTION
**Pull request by bashu22tiwari**

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { toastController } from '@ionic/vue';
+import { helperNameMap } from '@vue/compiler-core';
+import  Papa  from 'papaparse'
```
Comment (ymaheshwari1) [inline src/utils/index.ts:3]: Remove extra space.

File: src/utils/index.ts [hunk 2]
```diff
@@ -17,4 +19,66 @@ const showToast = async (message: string) => {
       return toast.present();
 }
 
-export { showToast, hasError }
+
```
Comment (ymaheshwari1) [inline src/utils/index.ts:22]: Remove the commented code, if not needed.

File: src/utils/index.ts (hunk 1 above)
Comment (bashu22tiwari) [reply src/utils/index.ts]: Removed the extra space.

File: src/utils/index.ts [hunk 3]
```diff
@@ -17,4 +19,66 @@ const showToast = async (message: string) => {
       return toast.present();
```
Comment (ymaheshwari1) [inline src/utils/index.ts:19]: Improve indentation.

File: src/utils/index.ts [hunk 4]
```diff
@@ -17,4 +19,66 @@ const showToast = async (message: string) => {
...
+
+
+
+const parseCsv = async (file: File, options: any) => {
+  return new Promise((resolve,reject) => {
+    Papa.parse( file, {
+      header: false,
+      skipEmptyLines: true,
+      complete: function( results: any ){
+          if(results.errors.length){
```
Comment (ymaheshwari1) [inline src/utils/index.ts:72]: Improve indentation.

File: src/utils/index.ts (hunk 3 above)
Comment (bashu22tiwari) [reply src/utils/index.ts]: Indentation improved.
Thank You for reviewing my code.

File: src/utils/index.ts (hunk 4 above)
Comment (bashu22tiwari) [reply src/utils/index.ts]: Indentation improved.
Thank You for reviewing my code.

**Comment by ymaheshwari1** [comment]
LGTM

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**